mattermost/desktop
# Use a Mattermost animation icon when loading a page in the app

Before submitting, please confirm you've

Please provide the following information:

**Summary**
Use a Mattermost animation icon when loading a page in the app.

It seems huge changes are needed to locate the icon at the vertical center for me. So I simply used 64px margin from the top of content area.

<img width="539" alt="loading" src="https://user-images.githubusercontent.com/1412057/32175198-8a8a7084-bdc7-11e7-9553-57795d0bf06e.png">

**Issue link**
#490

**Test Cases**
Normal case:
1. Open the app.
2. The animation icon should appear until the page has been loaded.

Error case:
1. Open the app.
2. Unplug LAN cable or disconnect from Wi-Fi.
3. Reload the app (Ctrl+Shift+R).
4. The error should appear.
5. Recover the internet connection.
6. The animation icon should appear on next reloading.

**Additional Notes**
https://circleci.com/gh/yuya-oc/desktop/456#artifacts
